src/main/app/src/components/SettingsPage.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Link, Switch, useLocation } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import DisplaySettings from './settingMiniPages/DisplaySettings';
+import UserManagement from './settingMiniPages/UserManagement';
+
+
+const useStyles = makeStyles((theme) => ({
+  menu: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    '& > *': {
+      margin: theme.spacing(1),
+      width: theme.spacing(30),
+      height: theme.spacing(25),
+    },
+  },
+  mainDisplay: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    '& > *': {
+      margin: theme.spacing(1),
+      width: theme.spacing(120),
+      height: theme.spacing(50),
+    },
+  },
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary,
+  },
+}));
+
+
+const SettingsMenu = () => {
+  const classes = useStyles();
+  const location = useLocation();
+  return (
+    <div className={classes.menu}>
+      <Paper elevation={3}>
+        <MenuList>
+          <Link to='/settings/userManagement'
+            selected={location.pathname === '/settings/userManagement'}
+            className={classes.link}
+          >
+            <MenuItem>
+              User Management
+                        </MenuItem>
```

Comment:
Just fix the formatting here so that the closing tag is aligned with the opening tag above, like:

```jsx
<MenuItem>
  User Management
</MenuItem>
```